plugins/package-managers/spdx/src/funTest/kotlin/SpdxDocumentFileFunTest.kt

```diff
@@ -175,6 +175,25 @@ class SpdxDocumentFileFunTest : WordSpec({
             }
         }
 
+        "retrieve nested dependson dependencies" {
```

Comment:
Nit: Let's use "DEPENDS_ON" instead of "dependson" here to resemble the enum name.